support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'preact/hooks';
+import { init as initAbTests } from 'helpers/abTests/abtest';
+import { config } from 'helpers/contributions';
+import { getSettings } from 'helpers/globalsAndSwitches/globals';
+import { detect, glyph } from 'helpers/internationalisation/currency';
+import { setProductType } from 'helpers/redux/checkout/product/actions';
+import { getContributionType } from 'helpers/redux/checkout/product/selectors/productType';
+import {
+	useContributionsDispatch,
+	useContributionsSelector,
+} from 'helpers/redux/storeHooks';
+import type { CheckoutNudgeProps } from './checkoutNudge';
+
+type CheckoutNudgeContainerProps = {
+	renderNudge: (props: CheckoutNudgeProps) => JSX.Element;
+};
+
+export function CheckoutNudgeContainer({
+	renderNudge: renderNudge,
+}: CheckoutNudgeContainerProps): JSX.Element | null {
+	const contributionType = useContributionsSelector(getContributionType);
+	const dispatch = useContributionsDispatch();
+	const { countryGroupId, countryId } = useContributionsSelector(
+		(state) => state.common.internationalisation,
+	);
+	const abParticipations = initAbTests(
```

Comment:
I don't think we should be calling `initAbTests` to get `abParticipations` they should be available in the Redux state in `state.common. abParticipations`.

Reply:
Ah I see it in Redux now, will do retrieval from Redux now

Reply:
For reference this is where we assign `abParticipations` it's value when setting up the initial Redux state: https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/redux/utils/setup.ts#L88.

Reply:
Completed - makes sense to use redux wherever possible, saves repeat getting of the same participations